include/ros1_bridge/factory.hpp

```diff
@@ -128,9 +130,10 @@ class Factory : public FactoryInterface
       return;
     }
 
-    std::string key = "callerid";
-    if (connection_header->find(key) != connection_header->end()) {
-      if (connection_header->at(key) == "/ros_bridge") {
+    const auto caller = connection_header->find("callerid");
```

Comment:
This stuff is need to allow for a random name for the bridge that makes it easier to identify